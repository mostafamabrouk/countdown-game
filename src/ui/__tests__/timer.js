/* eslint-disable no-undef */
import React from "react";
import renderer from "react-test-renderer";
import { Timer } from "../";
import "jest-styled-components";

test("Timer renders withouut any parameters", () => {
  const component = renderer.create(<Timer />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("Timer renders with minutes and seconds", () => {
  const component = renderer.create(<Timer minutes="00" seconds="05" />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
