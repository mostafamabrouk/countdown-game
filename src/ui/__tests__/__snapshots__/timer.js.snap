// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer renders with minutes and seconds 1`] = `
.c0 {
  opacity: 0.5;
  font-family: Helvetica;
  font-size: 48px;
  color: #00648f;
  text-align: center;
  -webkit-transition: 0.2s all ease;
  transition: 0.2s all ease;
  margin: 0 0 30px;
}

<div
  className="c0"
>
  00
  :
  05
</div>
`;

exports[`Timer renders withouut any parameters 1`] = `
.c0 {
  opacity: 0.5;
  font-family: Helvetica;
  font-size: 48px;
  color: #00648f;
  text-align: center;
  -webkit-transition: 0.2s all ease;
  transition: 0.2s all ease;
  margin: 0 0 30px;
}

<div
  className="c0"
>
  :
</div>
`;
